__tests__/integration.test.js

```diff
@@ -236,3 +236,102 @@ describe("postComment", () => {
       });
   });
 });
+describe('patchArticleByID', () => {
+  it('Should correctly update specified article when passed positive integer ', () => {
+    
+    return request(app)
+    .patch("/api/articles/5/")
+    .send({inc_votes: 2})
+    .expect(200)
+    .then((response) => {
+      expect(response.body.article).toMatchObject({
+        article_id: 5,
+        title: expect.any(String),
+        body: expect.any(String),
+        votes: 2,
+        topic: expect.any(String),
+        author: expect.any(String),
+        created_at: expect.any(String),
+      });
+    });
+  });
+  it('Should correctly update specified article when passed negative integer ', () => {
+    return request(app)
+    .patch("/api/articles/1/")
+    .send({inc_votes: -50})
+    .expect(200)
+    .then((response) => {
+      expect(response.body.article).toMatchObject({
+        article_id: 1,
+        title: expect.any(String),
+        body: expect.any(String),
+        votes: 50,
+        topic: expect.any(String),
+        author: expect.any(String),
+        created_at: expect.any(String),
+      });
+    });
+  });
+  it('Should correctly update specified article when passed negative integer ', () => {
+    return request(app)
+    .patch("/api/articles/5/")
+    .send({inc_votes: 0})
+    .expect(200)
+    .then((response) => {
+      expect(response.body.article).toMatchObject({
+        article_id: 5,
+        title: expect.any(String),
+        body: expect.any(String),
+        votes: 0,
+        topic: expect.any(String),
+        author: expect.any(String),
+        created_at: expect.any(String),
+      });
+    });
+  });
+  it('Should return 200 plus original object unchanged when no inc_votes is passed. ', () => {
+    return request(app)
+    .patch("/api/articles/5/")
+    .send()
+    .expect(200)
+    .then((response) => {
+      expect(response.body.article).toMatchObject({
+        article_id: 5,
+        title: expect.any(String),
+        body: expect.any(String),
+        votes: 0,
+        topic: expect.any(String),
+        author: expect.any(String),
+        created_at: expect.any(String),
+      });
+    });
+  });
```

Comment:
This should really be a 400 Bad Request since this endpoint is expecting an object.

Reply:
Cool!! Will do that now